deploy/docker-compose.yml

```diff
@@ -1,13 +1,17 @@
 version: '3'
 services:
   backoffice:
+    depends_on: 
+      - db
     build:
       context: ../apps/backoffice-v2/
       args:
         NPM_LOG_LEVEL: notice
     ports:
-      - ${BACKOFFICE_PORT}:3000
+      - ${BACKOFFICE_PORT}:5173
```

Comment:
Would be nice if the external port would also be an environment variable. :)

Reply:
change reverted, to the master state.